extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -1464,18 +1464,53 @@ describes.realWin('Events', {amp: 1}, env => {
       });
     });
 
-    describe('should create correct readyReportPromise', () => {
+    describe('should create correct reportReadyPromise', () => {
       it('with viewer hidden', () => {
         const stub = sandbox.stub(tracker.root, 'getViewer').callsFake(() => {
           return {
             isVisible: () => {return false;},
           };
         });
-        const promise = tracker.createReportReadyPromise_();
+        const promise = tracker.createReportReadyPromise_('hidden');
```

Comment:
Need to make this reflect the refactor to createReportReadyPromiseForHidden_(). (Done, currently linting on my machine, and will push in a few minutes.)

Reply:
Done